matrix_client/room.py

```diff
@@ -345,3 +346,22 @@ def add_room_alias(self, room_alias):
             return True
         except MatrixRequestError:
             return False
+
+    def repeat_last_messages(self, limit=10):
```

Comment:
Not too keen on the name. Repeat sounds more like repeating something we've already received. I'm more keen on "get_previous_messages" or "backfill". I believe the latter is what the JS people prefer.